src/app/exercise/runner/runner.component.html

```diff
@@ -1,3 +1,6 @@
-<slides-browser-window>
+<slides-browser-window *ngIf="!isConsole">
   <div #runner class="runner"></div>
 </slides-browser-window>
+<slides-console-window *ngIf="isConsole">
+  <div #runnerConsole class="runner"></div>
```

Comment:
Is this ever shown? It looks like `isConsole` is assigned to false and never reassigned.